### PR TITLE
Off2sdp

### PR DESCRIPTION
**Pull request by kerautret**
# PR Description
 a new tool to convert a mesh into a set of points (.sdp)

# Checklist

- [x] Doxygen documentation of the code completed (classes, methods, types, members...)
- [x] Check if it follows the tools structure described in [CONTRIBUTING.md](https://github.com/DGtal-team/DGtalTools-contrib/blob/master/CONTRIBUTING.md)
- [x] New entry in the [ChangeLog.md](https://github.com/DGtal-team/DGtalTools-contrib/blob/master/ChangeLog.md) added.
- [x] Update the readme with potentially a screenshot of the tools if it applies. 
- [x] No warning raised in Debug ```cmake``` mode (otherwise, Travis C.I. will fail).

### CODE REVIEW

File: geometry3d/off2sdp.cpp [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+using namespace std;
+using namespace DGtal;
+///////////////////////////////////////////////////////////////////////////////
+namespace po = boost::program_options;
+
+
+/**
+ @page off2sdp off2sdp
+ 
+ @brief  Converts a mesh into a set of points (.sdp). It can extract the mesh vertex (by default) or the center of faces.
```
Comment (dcoeurjo) [inline geometry3d/off2sdp.cpp]: the mesh vertices

File: geometry3d/off2sdp.cpp [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+  bool canStart = true;
+  if(! vm.count("input") || ! vm.count("output"))
+    {
+      trace.error() << " The input or output file name was not defined" << endl;
+      canStart = false;
+    }
+  if( !parseOK || vm.count("help")||argc<=1 || !canStart)
+    {
+      trace.info() << "Usage: " << argv[0] << " [input] [output] \n"
+                << "Converts a mesh into a set of points (.sdp). It can extract the mesh vertex (by default) or the center of faces.  \n"
```
Comment (dcoeurjo) [inline geometry3d/off2sdp.cpp]: vertices

**Comment by kerautret** [comment]
@dcoeurjo thanks done

**Comment by kerautret** [comment]
corrections done merging

**Pull request merged**